.github/workflows/ci.yml

```diff
@@ -41,6 +41,8 @@ jobs:
 
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v3
```

Comment:
You should update this to `v4` as well

Reply:
I tried v4 but there is an issue that prevents codecov from finding the coverage.xml file. It is a regression issue currently being looked at by the team https://github.com/codecov/codecov-action/issues/1278

Reply:
So for now I stick with v3 :)